wolfssl/wolfcrypt/asn.h

```diff
@@ -1173,6 +1173,8 @@ WOLFSSL_LOCAL int wc_OBJ_sn2nid(const char *sn);
 
 WOLFSSL_LOCAL int wc_EncodeName(EncodedName* name, const char* nameStr,
                                 char nameType, byte type);
+WOLFSSL_LOCAL int wc_EncodeName_cano(EncodedName* name, const char* nameStr,
```

Comment:
See comment about suggested name change to "wc_EncodeNameCanonical"

Reply:
The function name changed to "wc_EncodeNameCanonical"